lib/components/user/monitored-trip/trip-basics-pane.js

```diff
@@ -135,33 +132,50 @@ class TripBasicsPane extends Component {
           {/* Do not show trip status when saving trip for the first time
               (it doesn't exist in backend yet). */}
           {!isCreating && <TripStatus monitoredTrip={monitoredTrip} />}
-          <ControlLabel>Selected itinerary:</ControlLabel>
+          <ControlLabel>
+            <FormattedMessage id='components.TripBasicsPane.selectedItinerary' />
+          </ControlLabel>
           <TripSummary monitoredTrip={monitoredTrip} />
 
           <FormGroup validationState={errorStates.tripName}>
-            <ControlLabel>Please provide a name for this trip:</ControlLabel>
+            <ControlLabel>
+              <FormattedMessage id='components.TripBasicsPane.tripNamePrompt' />
+            </ControlLabel>
             {/* onBlur, onChange, and value are passed automatically. */}
             <Field as={FormControl} name='tripName' />
             <FormControl.Feedback />
-            {errors.tripName && <HelpBlock>{errors.tripName}</HelpBlock>}
+            {errors.tripName && (
+              <HelpBlock>
+                <FormattedMessage id={errors.tripName} />
```

Comment:
Please do not use a variable for a message ID. As noted in https://github.com/opentripplanner/otp-react-redux/pull/454/files#r709623526, you could use some kind of identifier to determine the error, but the message ID should still be a hard-coded string.